crates/ruff_python_ast/ast.toml

```diff
@@ -193,23 +203,25 @@ fields = [
   { name = "handlers", type = "ExceptHandler*" },
   { name = "orelse", type = "Stmt*" },
   { name = "finalbody", type = "Stmt*" },
-  { name = "is_star", type = "bool" },
+  { name = "is_star", type = "bool", skip_source_order = true },
```

Comment:
I'd be inclined to name the field `skip_visit` so that it can be shared between source order and the semantic order visitor. Or are there cases where we want to skip a field only in one but not the other (but maybe that's reason enough to hand write the visitor)